ppb/assets.py

```diff
@@ -92,33 +115,52 @@ def _draw_shape(self, renderer, rgb, **_):
         )
 
 
+def Square(r, g, b):
```

Comment:
Should we turn this into classes or just leave them as functions?

Reply:
Either way is fine with me.

I just wanted to keep this change backwards compatible and I want to document that it's basically just a wrapper of the "real" implementation.